ERROR - 2023-12-12 04:10:52 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'veterinaria', 3306, '', 0)
#1 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\Users\eliza\OneDrive\Documentos\Veterinaria\app\Controllers\Home.php(36): CodeIgniter\BaseModel->findAll()
#7 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\CodeIgniter.php(919): App\Controllers\Home->index()
#8 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\eliza\OneDrive\Documentos\Veterinaria\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\eliza\\...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\Users\eliza\OneDrive\Documentos\Veterinaria\app\Controllers\Home.php(36): CodeIgniter\BaseModel->findAll()
#6 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\CodeIgniter.php(919): App\Controllers\Home->index()
#7 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\eliza\OneDrive\Documentos\Veterinaria\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\eliza\\...')
#11 {main}
CRITICAL - 2023-12-12 04:10:52 --> Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` LIKE :nombre: ESCAPE \'!\'
AND  `contrasenia` LIKE :contrasenia: ESCAPE \'!\'', [...], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Home.php(36): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\eliza\\OneDrive\\Documentos\\Veterinaria\\public\\index.php')
CRITICAL - 2023-12-12 05:37:10 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 246.
 1 SYSTEMPATH\CodeIgniter.php(246): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(210): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 ROOTPATH\spark(88): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2023-12-12 05:37:10 --> Uncaught Error: Class "Locale" not found in C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\I18n\TimeTrait.php:74
Stack trace:
#0 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\I18n\TimeTrait.php(115): CodeIgniter\I18n\Time->__construct(NULL, NULL, NULL)
#1 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\HTTP\ResponseTrait.php(468): CodeIgniter\I18n\Time::now()
#2 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\HTTP\ResponseTrait.php(446): CodeIgniter\HTTP\Response->sendHeaders()
#3 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\Debug\ExceptionHandler.php(78): CodeIgniter\HTTP\Response->send()
#4 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Exceptions\FrameworkException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#5 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\FrameworkException))
#6 {main}
  thrown
【Previous Exception】
CodeIgniter\Exceptions\FrameworkException
The framework needs the following extension(s) installed and loaded: intl.
#0 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\CodeIgniter.php(246): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 C:\Users\eliza\OneDrive\Documentos\Veterinaria\system\CodeIgniter.php(210): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 C:\Users\eliza\OneDrive\Documentos\Veterinaria\spark(88): CodeIgniter\CodeIgniter->initialize()
#3 {main}
in SYSTEMPATH\I18n\TimeTrait.php on line 74.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
